docs/3-restructuredtext-basics.md

```diff
@@ -48,22 +48,20 @@ section that has some more information and a picture of a sphinx.
 
 Create a new section **above** the "Indices and tables" one:
 
-```
-Workspace
----------
+```rst
+Meet Sphinx
+-----------
 ```
 
 ### Image
 
-And now let's add a picture to our new section. You can use any image you like; we chose one we
-found online and downloaded from [unsplash](https://unsplash.com). Save your image in the
-`docs/_static` directory and type under your header:
+And now let's add a picture to our new section. You can use any image you like; we used [flickr](https://www.flickr.com/) and searched for `greek sphinx`. Save your image in the `docs/_static` directory and link it  under your header as follows:
```

Comment:
Flickr photos might have copyrights, let's use unsplash.

Reply:
I dont think the Egyptian sphinx has the same story about the riddle? It just guards the city 😅  and splash doesn't have  any photos of greek sphinx :( this is why i went to flickr I think the one I picked didn't have copyright issues but we can check again or reference it? maybe instruct people to reference it idk